lib/super_digit.rb

```diff
@@ -1,15 +1,28 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n+m)
 def super_digit(n)
```

Comment:
This works but because you're doing the summation within the recursion, the time complexity is O(d^2) where d is the number of digits.